.github/workflows/minimal-clang.yml

```diff
@@ -32,28 +32,36 @@ jobs:
 
         include:
           - b2_toolset: clang-3.9
-            b2_cxxstd: 03,11
+#            b2_cxxstd: 03,11
+            b2_cxxstd: 14
```

Comment:
I'd simply remove the commented lines to keep the YAML-s compact, easier to maintain.

Reply:
Done.